human_experiments/scripts/run_experiment

```diff
@@ -181,7 +182,16 @@ fi
 
 if [[ $GOTO -le 13 ]]; then
   echo " "
-  text="[13] Type [${EMPH}quit${NC}] to end the experiment and close "
+  yes_no_question "[13] Do you want to stop ${EMPH}Lab Recorder recordings and Kill the Lab Recorder Instances on CAT${NC}?"
```

Comment:
Same as the above comment (remove unnecessary titlecasing, change Lab Recorder to LabRecorder)

Reply:
I preferred not to have this as a yes/no question to avoid having us ending an experiment by mistake. Typing `quit` explicitly means we really want to end it. I'm okay with yes/no if everyone else agree with it.